scapy/contrib/ikev2.py

```diff
@@ -456,12 +754,7 @@ class IKEv2_payload_CERT(IKEv2_class):
 bind_layers( UDP,           IKEv2,        dport=500, sport=500) # TODO: distinguish IKEv1/IKEv2
 bind_layers( UDP,           IKEv2,        dport=4500, sport=4500)
 
-def ikev2scan(ip):
+def ikev2scan(ip, **kwargs):
+    """Send a IKEv2 SA to an IP and wait for answers."""
     return sr(IP(dst=ip)/UDP()/IKEv2(init_SPI=RandString(8),
-                                      exch_type=34)/IKEv2_payload_SA(prop=IKEv2_payload_Proposal()))
-
-# conf.debug_dissector = 1
-
-if __name__ == "__main__":
-    from scapy.main import interact
-    interact(mydict=globals(), mybanner="IKEv2 alpha-level protocol implementation")
+                                      exch_type=34)/IKEv2_payload_SA(prop=IKEv2_payload_Proposal()), **kwargs)
```

Comment:
To the line 759: https://github.com/secdev/scapy/pull/661/files#diff-487e064743c1edea916f1e38ec0175fbR759